fgl.cabal

```diff
@@ -1,5 +1,5 @@
 name:          fgl
-version:       5.5.2.3
+version:       5.5.2.4
```

Comment:
Please don't do this.

1) The third component should be the one being bumped as per the PVP, not the fourth.

2) This is my responsibility as maintainer (I might have other changes, etc.)